internal/cri/server/container_create.go

```diff
@@ -161,18 +161,24 @@ func (c *criService) CreateContainer(ctx context.Context, r *runtime.CreateConta
 		return nil, fmt.Errorf("failed to query sandbox platform: %w", err)
 	}
 
+	ociRuntime, err := c.config.GetSandboxRuntime(sandboxConfig, sandbox.Metadata.RuntimeHandler)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get sandbox runtime: %w", err)
+	}
+
 	var volumeMounts []*runtime.Mount
 	if !c.config.IgnoreImageDefinedVolumes {
```

Comment:
I think we have a latent configuration error here to take care of...

The config switch for ignoreimagedefinedvolumes not "extra" volume mounts passed through from the k8s pod+container spec.

Continuing along with latent configuration error now makes the error even more obvious.  

IMO this if/else block regarding filtering of the volumes defined in the image spec should not be filtering any of the extra volumes passed in from k8s.

Reply:
makes sense, thanks!